package.json

```diff
@@ -32,15 +32,15 @@
   "homepage": "https://github.com/fastify/fastify-mongodb#readme",
   "devDependencies": {
     "@fastify/pre-commit": "^2.0.2",
-    "@types/node": "^18.0.0",
-    "fastify": "^4.0.0-rc.3",
+    "@types/node": "^18.11.9",
+    "fastify": "^4.10.2",
     "standard": "^17.0.0",
-    "tap": "^16.1.0",
-    "tsd": "^0.24.1"
+    "tap": "^16.3.2",
+    "tsd": "^0.25.0"
   },
   "dependencies": {
-    "fastify-plugin": "^4.0.0",
-    "mongodb": "^4.0.1"
+    "fastify-plugin": "^4.3.0",
+    "mongodb": "^4.12.1"
```

Comment:
These changes are unnecessary.

Reply:
mongodb node drive makes perfect and being up to date with the plugin system does. I looked over https://www.mongodb.com/docs/drivers/node/current/compatibility/ and there is not a lot of mongo v3   versions anymore. 4.8.x drive is a compromise because it supports 6. I know that's what I am using in my environment so using 4.0.1 does not.

Reply:
Again: 
You can install mongo version 4.8.x driver in your project. fastify-mongodb will use than that version. 

All your changes are unnecessary.

Reply:
There is no mongodb npm package with version 5.0.0.

With your change, you force all users of @fastify/mongodb to use atleast version 4.8.0 and above.